charts/node-analyzer/templates/configmap-cspm-analyzer.yaml

```
@@ -0,0 +1,17 @@
+{{ if .Values.nodeAnalyzer.cspmAnalyzer.deploy }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "nodeAnalyzer.fullname" . }}-cspm-analyzer
+  labels:
+{{ include "nodeAnalyzer.labels" . | indent 4 }}
+data:
+{{- $env := "PRODUCTION" -}}
+{{ if .Values.nodeAnalyzer.cspmAnalyzer.debug }}
+  {{- $env = "DEVELOPMENT" -}}
+{{ end}}
+  environment: {{ $env }}
+  external_nats_url: {{ include "nodeAnalyzer.natsUrl" . }}
+  cluster_name: {{ required "A valid .Values.clusterName is required" .Values.clusterName }}
```

Please use the helper `nodeAnalyzer.clusterName` so that it will use the global clusterName if provided

```suggestion
  cluster_name: {{ required "A valid clusterName is required" (include "nodeAnalyzer.clusterName" .) }}
```